forgerock-core/src/test/java/org/forgerock/android/auth/AppPinAuthenticatorTest.kt

```diff
@@ -83,13 +89,4 @@ class AppPinAuthenticatorTest {
             Offset.offset(1000))
     }
 
-    @Test
-    fun testExists() {
```

Comment:
we canot test this now, because now its just looking the actual file exist ..and its mocked already

Reply:
I think we can implement the exists method for the overridden KeyStoreRepository (The one using byteArrayOutputStream) and check if the byte array has data

Reply:
i like that idea , we can do like byteArrayOutputStream.toByteArray().isNotEmpty(). Let me change that